content/docs/practices/naming.md

```diff
@@ -75,7 +75,7 @@ The list is not exhaustive.
 | Family | Base unit | Remark | 
 | -------| --------- | ------ |
 | Time   | seconds   |        |
-| Temperature | celsius | _celsius_ is preferred over _kelvin_ for practical reasons. |
+| Temperature | celsius | _celsius_ is preferred over _kelvin_ for practical reasons. _kelvin_ is acceptable in special cases like color temperature and scientific usage. |
```

Comment:
This contradicts the docs at https://prometheus.io/docs/instrumenting/writing_exporters/#naming 6th paragraph. The simplest fix would be replacing must with should, though I think it'd be better to expand that paragraph a little explaining where it can be okay to do things like this (i.e. extremely unlikely to end up mixing units down the line) rather than changing here.

"Scientific usage" is also a bit vague - does that include all environmental/weather monitoring for example? I'd leave things to the concrete example of monitor color temperature.

Reply:
Can we maybe have a category "Color temperature and Science"

Reply:
For such a relatively rare case, that seems a bit much. Science is also still a bit vague, e.g. I've attempted science many times with machine temperature (turns out it was dodgy RAM).

Reply:
It's not really a contradiction because _kelvin_ would be the proper base unit in those examples.  I changed the remark in this PR to use the word "base unit", although I'm torn if this formal correctness is really worth making the remark more verbose.

A verbose explanation when multiple base units are appropriate is IMHO more confusing than helpful, as the case is very rare (so far, temperature is the only one). It would probably just trigger people to shoehorn the case of their own favorite unit into this line of argument. I suggest to just leave it with temperature here. Should anyone ever bring up the temperature example as a reason to push another alternative base unit, we could still expand on the underlying reasoning.

I deliberately added the "scientific usage" because I think it's important. I kept it deliberately vague for the sake of brevity, given that this is merely inside a remark. I don't think there is any danger that anybody will feel compelled to convert the readings from their weather monitoring device into K. However, there is a danger that somebody that needs to calculate Boltzmann factors will feel compelled to use °C if we only mention the color temperature case.

In the hope to address your comments, I have extended the remark in my latest push. My problem with it is that the remark is now much longer without any practical impact. It is mostly there to address your rather formal concerns, which in my personal opinion have little impact on what people will actually do. However, I'm willing to take the leap of faith that this a longer remark is here more beneficial than harmful.

Reply:
This seems to be internally-self consistent now.

> It would probably just trigger people to shoehorn the case of their own favorite unit into this line of argument. 

That's my primary concern with all of this.